core-web/apps/dotcms-ui/src/app/portlets/dot-pages/dot-pages-listing-panel/dot-pages-listing-panel.component.ts

```diff
@@ -187,4 +187,11 @@ export class DotPagesListingPanelComponent implements OnInit, OnDestroy, AfterVi
         this.store.getPages({ offset: 0 });
         this.store.setSessionStorageFilterParams();
     }
+
+    scrollToTop(): void {
+        this.scrollElement?.scroll({
+            top: 0,
+            left: 0
+        });
```

Comment:
Why this is not happeing on the level where `dot-pages` lives? We can outoput the event here or even listen in the store if there is a page change (I think)

Reply:
We need to reset the scroll when a site switch happens and also we need to close the context menu from the favorite panel when a scroll happens.

```
 this.scrollElement = document.querySelector('dot-pages');

        this.scrollElement?.addEventListener('scroll', () => {
            this.closeContextMenu();
            this.tableScroll.emit();
        });

        this.dotSiteService.switchSite$.pipe(takeUntil(this.destroy$), skip(1)).subscribe(() => {
            this.store.getPages({ offset: 0 });
            this.scrollToTop(); // To reset the scroll so it shows the data it retrieves
        });
```

Reply:
I think I can move this logic to `dot-pages` and emit a page change, but I'm not sure of how to close the Table context menu on scroll.